internal/core/metadata/init.go

```diff
@@ -26,29 +26,19 @@ import (
 	"github.com/edgexfoundry/edgex-go/internal/pkg/errorconcept"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/coredata"
-	"github.com/edgexfoundry/go-mod-core-contracts/clients/notifications"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/types"
 )
 
 // Global variables
 var Configuration = &ConfigurationStruct{}
-var nc notifications.NotificationsClient
 var vdc coredata.ValueDescriptorClient
 
 // BootstrapHandler fulfills the BootstrapHandler contract and performs initialization needed by the metadata service.
 func BootstrapHandler(wg *sync.WaitGroup, ctx context.Context, startupTimer startup.Timer, dic *di.Container) bool {
 
 	// initialize clients required by service.
 	registryClient := container.RegistryFrom(dic.Get)
-	nc = notifications.NewNotificationsClient(
-		types.EndpointParams{
-			ServiceKey:  clients.SupportNotificationsServiceKey,
-			Path:        clients.ApiNotificationRoute,
-			UseRegistry: registryClient != nil,
-			Url:         Configuration.Clients["Notifications"].Url() + clients.ApiNotificationRoute,
-			Interval:    Configuration.Service.ClientMonitor,
-		},
-		endpoint.Endpoint{RegistryClient: &registryClient})
```

Comment:
You've removed the instantiation of the notifications client.  You've created the helpers (in containers/notifications.go) to provide a unique name and getter for the notifications client in the dependency injection container.  You haven't actually added the service to the dependency injection container.  This code will compile and run but when you invoke an endpoint that requires the notification client, the service will panic.

This mirrors what you needed to do (and ultimately did) in a prior PR for the errorHandler.

Reply:
@michaelestrin Good catch, thanks. I need to run integration tests more often, too.